airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/metadata.py

```diff
@@ -127,14 +130,14 @@ def build_metadata(merged_df):
         catalogs = compute_catalog_overrides(merged_df)
         metadata["data"]["catalogs"] = catalogs
 
-        return metadata
+        return PartialMetadataDefinition.construct(**metadata)
```

Comment:
```suggestion
        return PartialMetadataDefinition(**metadata)
```
Just wondering why we're not building the model directly

Reply:
Oh I think its because you explicitly want to delay the validation in a separate step.

Reply:
Yup exactly that, these aren't "valid" until overrides are applied